### DISCUSSION TITLE
notes/%E8%BF%90%E7%BB%B4%E7%9B%B8%E5%85%B3%E8%AE%B0%E5%BD%95/Nginx

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#446] [Announcements]
# notes/%E8%BF%90%E7%BB%B4%E7%9B%B8%E5%85%B3%E8%AE%B0%E5%BD%95/Nginx

https://deantook.top/notes/%E8%BF%90%E7%BB%B4%E7%9B%B8%E5%85%B3%E8%AE%B0%E5%BD%95/Nginx.html

**Comment by deantook** [comment]
写的太好啦🥹

**Reply by Mister-Hope** [reply]
# DO NOT USE MY REPO IN  PRODUCTION!
